docs/markdown/platforms/AWS.md

```diff
@@ -65,6 +65,7 @@ titled ("Step 1: Choose an Amazon Machine Image (AMI)").
 ![image](aws_images/AWS_AMI.png)
```

Comment:
Is this functional?

Reply:
I want to leave this in for us in case we look at this again. I don't think there aren't any links to it in the documentation but if you think it is too obvious, we can hide it somewhere else. We can also put a notice at the top indicating that this is currently not supported.